api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/policy/ifaces/Policy.java

```diff
@@ -0,0 +1,28 @@
+/* $This file is distributed under the terms of the license in LICENSE$ */
+
+package edu.cornell.mannlib.vitro.webapp.auth.policy.ifaces;
+
+import edu.cornell.mannlib.vitro.webapp.auth.requestedAction.AuthorizationRequest;
+
+/**
+ * Represents the process of mapping an identifier that represents a user or principle and a action they are requesting
+ * to true, representing authorized or false, representing unauthorized.
+ *
+ * @author bdc34
+ *
+ */
+public interface Policy {
+    public PolicyDecision decide(AuthorizationRequest ar);
+
+    public default String getUri() {
+        return "";
+    }
+
+    public default long getPriority() {
+        return 0;
+    }
+
+    public default String getShortUri() {
+        return "";
+    }
```

Comment:
I wouldn't use default interface methods when creating a new interface as they were primarily introduced to allow us to add new methods to an interface that are automatically available in the implementations, in order to perserve backward compatibility. I would personally use the abstract class here. On the other hand, you surely have more experience with this, I would like to hear your opinion on doing it this way.

Reply:
It is not a new interface. It was done for the reason you described: add new methods to an interface that are automatically available in the implementations.

Reply:
Then it is the correct use-case. It was marked green for the whole class so I figured that everything was added during this PR. Great job for implementing it this way! We can maybe consider switching to an abstract class if this interface does not have many implementations but, personally, I wouldn't bother 😄.